### PR TITLE
initial work on vehicles

### PR DESCRIPTION
**Pull request by aap**
~~SNAFU~~ not so SNAFU anymore

### CODE REVIEW

File: src/vehicles/Automobile.cpp [hunk 1]
```diff
@@ -910,448 +902,73 @@ CAutomobile::ProcessControl(void)
...
+			   (m_aSuspensionSpringRatio[1] < 1.0f || m_aSuspensionSpringRatio[3] < 1.0f))
+				ApplyTurnForce(-GRAVITY*Min(m_fTurnMass, 2500.0f)*GetUp(), -1.0f*GetForward());
 		}
+*/
+
+		float steerRange;
+		if(fwdSpeed > 0.01f && m_aWheelTimer[CARWHEEL_FRONT_LEFT] > 0.0f && m_aWheelTimer[CARWHEEL_FRONT_RIGHT] > 0.0f && GetStatus() == STATUS_PLAYER){
+			CColPoint point;
+			point.surfaceA = SURFACE_WHEELBASE;
+			point.surfaceA = SURFACE_TARMAC;
```
Comment (jte) [inline src/vehicles/Automobile.cpp:931]: surfaceB?

File: src/vehicles/Vehicle.cpp [hunk 2]
```diff
@@ -867,15 +871,21 @@ CVehicle::ProcessWheel(CVector &wheelFwd, CVector &wheelRight, CVector &wheelCon
 
 	float speedSq = sq(right) + sq(fwd);
 	if(sq(adhesion) < speedSq){
+//	if(0){
```
Comment (erorcun) [inline src/vehicles/Vehicle.cpp:874]: is this intended?

File: src/vehicles/Automobile.cpp [hunk 3]
```diff
@@ -891,16 +861,38 @@ CAutomobile::ProcessControl(void)
...
+		float acceleration = pHandling->Transmission.CalculateDriveAcceleration(m_fGasPedal, m_nCurrentGear, m_fChangeGearTime, fwdSpeed,
+					&m_fEngineInertiaVar1, &m_fEngineInertiaVar2, m_nDriveWheelsOnGround, gripCheat);
+//		float acceleration = pHandling->Transmission.CalculateDriveAcceleration(m_fGasPedal, m_nCurrentGear, m_fChangeGearTime, fwdSpeed, gripCheat);
+		acceleration /= m_fForceMultiplier;
+
+		if(IsRealHeli() || IsRealPlane())
+			acceleration = 0.0f;
+
+		if(Abs(acceleration) > 0.0f)
+			m_fEngineWork += Abs(acceleration);
```
Comment (erorcun) [inline src/vehicles/Automobile.cpp:878]: is this correct?

Comment (aap) [reply src/vehicles/Automobile.cpp]: better name now, but yes.

**Pull request merged**